### ISSUE TITLE
[BUG] A problem with the "Don't automatically filter guides by user language" option

### ISSUE THREAD

**Issue by NarciSource** [IsThereAnyDeal/AugmentedSteam#1596]
###  Behavior
[View All] button in the Guides tab does not work.

###  Problem
There is a problem with the FRemoveGuidesLangFilter function.
This function includes '.guides_home_view_all_link > a' in its scope shown in the second paragraph.
The href value of this node is a JavaScript function. So, The problem was caused by adding a parameter to the end of the function.
I don't know why this node was included.
Anyway, shouldn't it be removed?

**Comment by candela97** [comment]
Yeah, the first View All link might be for showing hidden categories and we shouldn't be touching that. The selector is still needed to replace the link for viewing all popular Steam guides though.

**Comment by NarciSource** [comment]
Thanks for the reply

**Comment by candela97** [comment]
Uh, this'll be fixed by the linked PR, but it's not merged yet
